docs/guides/email/02-basic-email-system.md

```diff
@@ -678,16 +747,20 @@ service auth-worker {
 ...
 ```
 
-OK, use the command to start your service-- `systemctl start dovecot.service`
+Next enable Dovecot:
+
+```bash
+Shell(192.168.100.7) > systemctl enable --now dovecot
+```
 
 !!! info 
```

Comment:
```suggestion
!!! info "Info"
```

Reply:
We have discussed this before @SergeCroise, but adding the "Info" and "Warning" to custom titles isn't required. In fact, I think we discussed that translators could decide if they wanted to add the translated title to the admonition themselves. If I recall, that was the consensus decision, wasn't it?